pkg/ebpf/ingress_node_firewall_loader.go

```diff
@@ -480,7 +480,7 @@ func BuildEBPFKey(ifID uint32, cidr string) (BpfLpmIpKeySt, error) {
 		copy(key.IpData[:], ip.To16())
 	}
 	pfLen, _ := ipNet.Mask.Size()
-	key.PrefixLen = uint32(pfLen)
+	key.PrefixLen = uint32(pfLen) + 32
```

Comment:
Add a comment here that explains why we add 32 to the prefixlen

Reply:
removed magic number and added const with comment